src/components/bottom_bar_statistic/aggregate_statistics_store.ts

```diff
@@ -107,12 +107,17 @@ export class AggregateStatisticsStore extends SpreadsheetStore {
     const sheetId = getters.getActiveSheetId();
     const cells = new Set<EvaluatedCell>();
 
+    const isColHidden = memoize((col: number) => getters.isColHidden(sheetId, col));
+    const isRowHidden = memoize((row: number) => getters.isRowHidden(sheetId, row));
```

Comment:
does it really makes a significant difference ?
I believe the benchmark in the commit message is not the correct one. It's suspiciously the same as the previous commit

Reply:
And if this getter really cause performance issue (which I'm not sure how since it really does nothing), shouldn't it be memoized inside the plugin rather than here ? That way every call to this getter in other plugin/components is faster

Reply:
I don't know what to say. Don't believe me? then try it for yourself. 
The reason I haven't set it everywhere right now is because I don't  want to have to invalidate the cache all the time and it's pretty much the only place where it's heavily used.

Reply:
Sorry I'm a moron and can't make proper commit messages apparemently. The gain was around 30° ms. I did another round of benchmark and will update the values accordingly

Reply:
The time is mostly spent in isColFolder and isRowFolder (of header_grouping.ts). 
I think it would be wise to store on this plugin the list of cols and rows that are folded by sheet, so these getters becomes immediate.
![image](https://github.com/odoo/o-spreadsheet/assets/7260798/4254911a-5082-42f2-acde-82c0de6983e7)